admin/config/environments/production/database.json

```diff
@@ -0,0 +1,18 @@
+{
+  "defaultConnection": "default",
+  "connections": {
+    "default": {
+      "connector": "bookshelf",
+      "settings": {
+        "client": "postgres",
+        "database": "d80rgd499a8c8j",
+        "host": "ec2-3-231-46-238.compute-1.amazonaws.com",
+        "port": "5432",
+        "username": "qiikutniibytvn",
+        "password": "76305121d96bfc512d13b1af5d3cedcca44d40ac11db457cfb9abf5b9b8f8e5a",
```

Comment:
we should probably put these database values into environment variables

Reply:
https://strapi.io/documentation/3.0.0-alpha.x/configurations/configurations.html#dynamic-configurations